api/app/src/main/kotlin/packit/model/Packet.kt

```diff
@@ -0,0 +1,19 @@
+package packit.model
+
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "packet")
+data class Packet(
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
These ids will always come from `outpack` and be Strings, so I don't think we want this `GeneratedValue` attribute